src/app/geo-map/map-binding-data-csv/map-binding-data-csv.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
```

Comment:
do we really need .spec.ts file for our samples? Is this a new requirement?

Reply:
all will be removed in my latest commit